RUFAS/output_manager.py

```diff
@@ -138,6 +139,13 @@ def __init__(self) -> None:
             self.saved_pool_chunks_num: int = 0
             self.saved_pool_chunks_path: Path | None = None
 
+            self.available_memory: int = 0
+            self.average_add_variable_call_addition: int | None = None
```

Comment:
```suggestion
            self.average_add_variable_call_addition: int = 0
```
I think it is better to have this default to an int, I got a NoneType error when trying to run it with the default set to None.